s4e-backend/src/scripts/seed.sh

```diff
@@ -106,6 +106,17 @@ for schema_path in $(ls $configuration_path/schema); do
   unset name type content payload
 done
 
+# Create categories
+echo "Creating categories"
+url=$api_endpoint/api/admin/products/categories
+curl --silent --include --url $url \
+  --request POST \
+  --header "Content-Type: application/json" \
+  --header "Authorization: Bearer $auth_token" \
+  --data "{}"
```

Comment:
I don't think you have to specify this (i.e. `--data "{}"`), other POST requests with an empty request body in this script work fine.